src/main/java/duke/command/DoneCommand.java

```diff
@@ -0,0 +1,27 @@
+package duke.command;
+
+import duke.utils.DukeException;
+import duke.utils.DukeFileHandler;
+import duke.utils.TaskList;
+import duke.utils.Ui;
+
+
+public class DoneCommand extends Command {
+    private final int entryDone;
```

Comment:
Similar to my previous comment in the DeleteCommand class, perhaps we can explore adopting a more specific variable name here as well! 😺